.changeset/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@1.7.0/schema.json",
-  "changelog": ["@changesets/changelog-github",
```

Comment:
can/should this be removed from the `package.json` at the project's root?

Reply:
yeah, didn't realise it was a different package from `@changeset/cli`
removed now in https://github.com/hashicorp/design-system/pull/1740/commits/e71fca6fcf5ac55308efef8ffb56a0534f944207

Reply:
We found out we removed too much: https://github.com/hashicorp/design-system/pull/1743